### PR TITLE
Updated docs for 6.x

### PR DESCRIPTION
**Pull request by justin808**

### CODE REVIEW

**Comment by gauravtiwari** [comment]
Thanks, @justin808 for this. Is it possible to keep the `webpacker-react-on-rails` section on react on rails repo and then we can link to it from Webpacker? Just want to keep the Readme slim :)

**Comment by justin808** [comment]
@gauravtiwari I updated per your request.

File: README.md [hunk 1]
```diff
@@ -256,13 +273,16 @@ const { webpackConfig } = require('@rails/webpacker')
 
 console.log(webpackConfig.output_path)
 console.log(webpackConfig.source_path)
+
+// Or to print out your whole webpack configuration
+console.log(JSON.stringify(webpackConfig, undefined, 2)
```
Comment (n-rodriguez) [inline README.md:278]: There is a typo here. It should be `console.log(JSON.stringify(webpackConfig, undefined, 2))` (missing closing bracket)

**Review by n-rodriguez — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (justin808) [reply README.md]: @n-rodriguez Thanks! Fixed!

**Comment by justin808** [comment]
> Thanks, @justin808 for this. Is it possible to keep the `webpacker-react-on-rails` section on react on rails repo and then we can link to it from Webpacker? Just want to keep the Readme slim :)

I moved all the old React docs to: https://github.com/shakacode/react_on_rails/blob/master/docs/rails-webpacker-react-integration-options.md

**Comment by justin808** [comment]
@gauravtiwari This is ready for review again. I rebased on master.

**Comment by justin808** [comment]
@gauravtiwari I rebased my doc changes. Can you please merge or give me feedback?

**Pull request merged**